lib/fastlane/plugin/wpmreleasetoolkit/helper/ios/ios_version_helper.rb

```diff
@@ -101,6 +194,11 @@ def self.bump_version_release
           return verified_version
         end
 
+        # Updates the app_version entry in the `Deliverfile`
+        #
+        # @param [String] new_version The new value to set the `app_version` entry to.
+        # @raise [UserError] If the Deliverfile was not found.
+        #
         def self.update_fastlane_deliver(new_version)
           fd_file = "./fastlane/Deliverfile"
```

Comment:
This path seems mildly fragile, but not necessarily the job of this PR to fix it.